plugin_runner/sandbox.py

```diff
@@ -42,6 +42,7 @@
         "canvas_sdk.handlers",
         "canvas_sdk.protocols",
         "canvas_sdk.utils",
+        "canvas_sdk.v1",
```

Comment:
I'm not sure how anyone is testing this stuff without this line, but I couldn't get anything to work in `compute` without this line.

Do we want `v1` to be the module allowed, or go one more down to `data`?

Reply:
You got me, I was running the code in the python repl and not in a plugin. Great catch.